### DISCUSSION TITLE
Recommended system specifications

### DISCUSSION THREAD

**Discussion by loxiry** [lavalink-devs/Lavalink#734] [Q&A] [answered]
I want to use Lavalink for my discord bot. But I have no idea how much resources it uses. How many resources does it use?

**Comment by devoxin** [comment, marked as answer]
There is no fixed specification as it's heavily dependent on your usage. For example, if you intend to only serve a single guild, this implies you'll only ever have a single stream, in which case the hardware requirements are quite low. This is further complicated by the source format, for example if you're playing opus (which requires no transcoding), pcm (which only requires encoding), or mp3 (which requires transcoding). Even after knowing all this, we can't really give you a definite answer, only a rough ballpark. If you were to specify a rough configuration and usage of Lavalink, we may be able to offer better feedback.
